common/src/main/java/com/wynntils/services/itemfilter/ProfessionItemFilter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter;
+
+import com.wynntils.models.items.WynnItem;
+import com.wynntils.models.items.items.game.GatheringToolItem;
+import com.wynntils.models.items.items.game.IngredientItem;
+import com.wynntils.models.items.items.game.MaterialItem;
+import com.wynntils.models.profession.type.ProfessionType;
+
+public class ProfessionItemFilter implements ItemFilter {
+    private ProfessionType profession;
+
+    public ProfessionItemFilter(ProfessionType profession) {
+        // TODO FILTER NULL
```

Comment:
What does this mean :)?

Reply:
Oops my apologies, I forgot to remove this.